src/guide/release-notes.md

```diff
@@ -67,6 +67,30 @@ The following are general practices that should be followed when creating releas
   * Background info/context
   * Example: A new system property, `jdk.disableLastUsageTracking`, has been introduced to disable JRE last usage tracking for a running VM.
 
+## Linking options within the release note
+
+As well as the standard [CommonMark link types](https://spec.commonmark.org/0.31.2/#links) the following custom links are supported:
+
+**Linking of JBS issues**
+
+  * any JBS ID of the form `JDK-NNNNNNN` will be linked
+
+**Linking to the Java documentation**
+
+  * the linking options defined in JavaDoc [MarkDown](https://openjdk.org/jeps/467#Links) are supported.
+  * links to the JDK tools is supported. To differentiate between `[JarSigner]` the class, and `[jarsigner]` the tool, the tool reference should be in all lowercase.
```

Comment:
I think it would be nice if "JDK tools" was a link to the tools page. I know it's version specific, but the JDK 25 page should be good enough for a few years :-)